app/views/logs/_log_list.html.erb

```diff
@@ -3,9 +3,9 @@
   <div class="govuk-grid-column-three-quarters">
     <%= render(SearchResultCaptionComponent.new(searched:, count: pagy.count, item_label:, total_count:, item: "logs", filters_count: applied_filters_count(@filter_type))) %>
     <% if logs&.any? %>
-      <%= govuk_link_to "Download (CSV)", csv_download_url, type: "text/csv", class: "govuk-!-margin-right-4" %>
+      <%= govuk_link_to "Download (CSV)", csv_download_url, type: "text/csv", class: "govuk-!-margin-right-4", style: "white-space: nowrap" %>
       <% if @current_user.support? %>
-        <%= govuk_link_to "Download (CSV, codes only)", csv_codes_only_download_url, type: "text/csv", class: "govuk-!-margin-right-4" %>
+        <%= govuk_link_to "Download (CSV, codes only)", csv_codes_only_download_url, type: "text/csv", class: "govuk-!-margin-right-4", style: "white-space: nowrap" %>
```

Comment:
Ah, sorry, so check with design, but I think we deliberately wanted all the links to be on a new line if the filter is applied https://dluhcdigital.atlassian.net/browse/CLDC-2743

Reply:
you're absolutely right - I've fixed the spacing issue so now we don't have that indentation I mentioned above that makes it look weird when it's on a separate line: 
![image](https://github.com/communitiesuk/submit-social-housing-lettings-and-sales-data/assets/94526761/f0a586c0-5a16-40b6-8aa0-36be8bb2105e)
![image](https://github.com/communitiesuk/submit-social-housing-lettings-and-sales-data/assets/94526761/2864bf8b-e0ba-4bd9-a5b3-534e233b24b6)
![image](https://github.com/communitiesuk/submit-social-housing-lettings-and-sales-data/assets/94526761/32efca33-ed1a-49a8-b8cf-5868cc776fa5)
![image](https://github.com/communitiesuk/submit-social-housing-lettings-and-sales-data/assets/94526761/84c3f88a-9b98-4fbd-9e60-61b4b121cb37)

Reply:
these are all the new behaviours ^ (this is the indentation issue that used to be present:)
<img width="862" alt="image" src="https://github.com/communitiesuk/submit-social-housing-lettings-and-sales-data/assets/94526761/d220c762-f3c7-4311-812e-38d752c0e547">